semantics.md

```diff
@@ -64,7 +64,7 @@ Square brackets form a list constructor, which expects zero or more expressions
 
 Braces form a begin-expression, which evaluates all but the last expression for side effects only and produces the last expression's value.
 
-An on-expression, as a side effect, creates an internal-use advancement that, when triggered (according to the expression's trigger), evalues the given commands. It produces the namespaced id of that internal-use advancement.
+An on-expression, as a side effect, creates an internal-use advancement that, when triggered (according to the expression's trigger), evaluates the expressions defined inside of it. It produces the namespaced id of that internal-use advancement.
```

Comment:
I think this just needs modification to the literal command semantics - a literal command can either be a string or a list of strings - if it's a list, it's expanded as if multiple strings were given

Reply:
Then again - this is probably just clarifying my bad wording